<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TRUNEXA-00023" failures="0" tests="2" name="Default test" time="0.035" errors="0" timestamp="2020-12-08T14:04:17 IST">
  <testcase classname="com.CRM.qa.TestCases.LoginPageTest" name="@BeforeTest setup" time="0.006">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.base/java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936)
at java.base/java.util.Properties.getProperty(Properties.java:1114)
at com.CRM.qa.Base.TestBase.Intialization(TestBase.java:46)
at com.CRM.qa.TestCases.LoginPageTest.setup(LoginPageTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.beforeRun(TestRunner.java:607)
at org.testng.TestRunner.run(TestRunner.java:578)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="com.CRM.qa.TestCases.LoginPageTest" name="@AfterTest TearDownt" time="-1.607416457482E9">
    <skipped/>
  </testcase> <!-- @AfterTest TearDownt -->
  <testcase classname="com.CRM.qa.TestCases.LoginPageTest" name="LoginPageTitleTest" time="0.0">
    <skipped/>
  </testcase> <!-- LoginPageTitleTest -->
  <testcase classname="com.CRM.qa.TestCases.LoginPageTest" name="LoginTest" time="0.0">
    <skipped/>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- Default test -->
